### ISSUE TITLE
Random position of console window

### ISSUE THREAD

**Issue by MartinCapousek** [ForkIssues/TrackerWin#1402]
Would be really helpful if the app would remember position/size of console window (probably before it is closed).

**Comment by DanPristupov** [comment]
The initial window position is defined by Windows. If you press `Cmd+R` and run `cmd` or `explorer` multiple times, you will see that they are positioned in `cascade` way.

**Comment by MartinCapousek** [comment]
Yeah, that's probably default behavior on how is Windows positioning newly created windows.
But as developer you should be able to set position & size of those windows.
